generators/lib/distros/octane.js

```diff
@@ -31,21 +32,29 @@ function init() {
     done();
   };
 
-  module.modifyComposer = function(yo, options, localComposer, isNewProject, done) {
+  module.loadComposer = function(yo, options) {
+    var composer = {};
+    var file = yo.templatePath('drupal/drupal/' + options.drupalDistroRelease + '/composer.json');
+    if (gadget.fsExistsSync(file)) {
+      composer = yo.fs.readJSON(file);
+      // Octane gets it's version of Drupal Core from Lightning.
+      // To minimize conflicts with the rest of the generator process, we defer
+      // removing this until the last time Octane can reasonably impose this logic.
+      delete composer.require['drupal/core'];
```

Comment:
Since this is the only difference between the Octane and the Drupal version, I wonder if this is the correct spot.  Can't we make this change in "modifyComposer" (esp since this is modifying the composer and not just "loading" it)